<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1679</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Machine Learning Implementation</string>
   </property>
  </widget>
  <widget class="QFrame" name="ML_frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>441</width>
     <height>711</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_choose_model">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browse_file_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_file">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Folder</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>File Name</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="RF_train_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Train Random Forest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confusion_matrix_btn_KNN">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot KNN Confusion Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="KNN_train_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Train KNN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CNN_train_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Train CNN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confusion_matrix_btn_CNN">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot CNN Confusion Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confusion_matrix_btn_RF">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot RF Confusion Matrix</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="plot_frame">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>1181</width>
     <height>791</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="rf_plot"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>371</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="knn_plot"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>361</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="cnn_plot"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
